### ISSUE TITLE
hsahlib blake2b and blake2s errors

### ISSUE THREAD

**Issue by CodeAlkemist** [PyCQA/pylint#1968]
### Steps to reproduce
1. Use python 3.6
2. add
```python
from hashlib import blake2b
```
to any file and try pylint
3. It will give the error that the module don't exists

### Current behavior
Despite being a python standard library function blake2 hashing classes are being reported as non existing by pylint.

### pylint --version output
```
(env) PS C:\Users\[omited]> pylint.exe --version
No config file found, using default configuration
pylint 1.8.3,
astroid 1.6.2
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)]
```
